app/utils/SemVer.scala

```diff
@@ -21,10 +21,10 @@ object SemVer {
   }
 
   case class SemVerVersion(
-                      major: Option[Int] = None,
```

Comment:
Instead of making `major` a `String` I wonder if we should add a `prefix: Option[String]`. I think that might make this a bit cleaner.  WDYT?